File: threads-query-app/src/main/java/connection/ConnectionClass.java
package connection;

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;

public class ConnectionClass {

    public static Connection getConnection() {
        Connection conn = null;
        final String DRIVER = "org.postgresql.Driver";
        final String URL = "jdbc:postgresql://192.168.1.120/postgres?currentSchema=henrique_northwind";
        final String USER = "henrique";
        final String PASSWORD = "1234";

        try{
            Class.forName(DRIVER);
            conn = DriverManager.getConnection(URL, USER, PASSWORD);
        }
        catch (SQLException | ClassNotFoundException e) {
            e.printStackTrace();
        }
        return conn;
    }
}

File: threads-query-app-kt/src/main/kotlin/connection/ConnectionClass.kt
package connection

import java.sql.Connection
import java.sql.DriverManager
import java.sql.SQLException

class ConnectionClass {

    companion object {
        private lateinit var conn: Connection
        private const val driver = "org.postgresql.Driver";
        private const val URL = "jdbc:postgresql://192.168.1.120/postgres?currentSchema=henrique_northwind"
        private const val USER = "henrique"
        private const val PASSWORD = "1234"

        fun getConnection(): Connection {
            try{
                Class.forName(driver)
                conn = DriverManager.getConnection(URL, USER, PASSWORD)
            }
            catch (se: SQLException){
                se.printStackTrace()
            }
            return conn
        }
    }
}